bundle/jsky.elevation.plot/src/main/java/jsky/plot/ElevationPanel.java

```diff
@@ -76,6 +77,23 @@
     // Color used to display elevation constraints
     private static final Color CONSTRAINT_COLOR = new Color(0.0F, 1.0F, 0.0F, 0.5F);
 
+    private static final Paint TIMING_WINDOW_PAINT;
+
+    static {
+        final BufferedImage bi = new BufferedImage(4, 4,  BufferedImage.TYPE_4BYTE_ABGR);
+        final int rgb = new Color(164, 160, 203).getRGB();
```

Comment:
Right, this defines how the area outside of the defined timing windows will be painted.